internal/controller/postgrescluster/pgbackrest_test.go

```diff
@@ -1419,6 +1432,16 @@ func TestReconcileManualBackup(t *testing.T) {
 					}
 					assert.Assert(t, foundImagePullSecret)
 
+					// verify the priority class
+					var foundPriorityClass bool
+					for _, job := range jobs.Items {
+						if job.Spec.Template.Spec.PriorityClassName == "some-priority-class" {
+							foundPriorityClass = true
+							break
+						}
+					}
+					assert.Assert(t, foundPriorityClass)
```

Comment:
⛏️ This loop and the one above it don't seem necessary. This assertion passes when at least one job has the priority class, and we've already asserted that `len(jobs.Items) == 1` on L1411.

If I understand correctly, this loop can be replaced with a single assertion:

```go
assert.Equal(t, jobs.Items[0].Spec.Template.Spec.PriorityClassName, "some-priority-class")
```

Reply:
:+1: updated.